diff --git a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootView.kt b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootView.kt
index 61c1b4c..c70ddc9 100644
--- a/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootView.kt
+++ b/node_modules/react-native-gesture-handler/android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerRootView.kt
@@ -38,9 +38,7 @@ class RNGestureHandlerRootView(context: Context?) : ReactViewGroup(context) {
     } else super.dispatchTouchEvent(ev)
 
   override fun dispatchGenericMotionEvent(event: MotionEvent) =
-    if (_enabled && rootHelper!!.dispatchTouchEvent(event)) {
-      true
-    } else super.dispatchGenericMotionEvent(event)
+    super.dispatchGenericMotionEvent(event)
 
   override fun requestDisallowInterceptTouchEvent(disallowIntercept: Boolean) {
     if (_enabled) {
